frontend/src/components/Town/interactables/Chess/ChessArea.tsx

```diff
@@ -79,14 +169,13 @@ function ChessArea({ interactableID }: { interactableID: InteractableID }): JSX.
  */
 export default function ChessAreaWrapper(): JSX.Element {
   const gameArea = useInteractable<GameAreaInteractable>('gameArea');
+  console.log(gameArea);
```

Comment:
Was this for debugging? If so can we comment this out or remove completely?

Reply:
yes, it was for debugging. I missed one while i was removing console logs.